__tests__/article_comments.test.js

```diff
@@ -0,0 +1,54 @@
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const articleData = require("../db/data/test-data/index");
+const userData = require("../db/data/test-data/index");
+const request = require("supertest");
+const app = require("../app");
+const sorted = require("jest-sorted");
+
+beforeEach(() => {
+  return seed(articleData, userData);
+});
```

Comment:
you're passing two arguements here which are essentially the exact same pieces of data (check your requires) you only need to pass the entire test data object, which is what both of the variables actually are. Rename line 3 articleData > testData, remove line 4, and then just pass testData as the arguement, this is misleading and confusing